wp-content/plugins/woodstock-extensions/framework/extensions/wbc_importer/demo-data/watch/theme-options.txt
{"last_tab":"1","tdl_maincontent_width":"1440","tdl_layout_type":"fullwidth","tdl_background_type":"custom_back","tdl_background_color":"#333333","tdl_background_img":{"url":"http://watch:8888/wp-content/uploads/2016/03/watch-bg-main-6.jpg","id":"1760","height":"1199","width":"1920","thumbnail":"http://watch:8888/wp-content/uploads/2016/03/watch-bg-main-6-150x150.jpg"},"tdl_background_repeat":"0","tdl_site_logo_noretina":{"url":"http://woodstock.temashdesign.com/watch/wp-content/uploads/sites/5/2016/03/woodstock-watch-logo.png","id":"1864","height":"75","width":"247","thumbnail":"http://woodstock.temashdesign.com/watch/wp-content/uploads/sites/5/2016/03/woodstock-watch-logo-150x75.png"},"tdl_site_logo_retina":{"url":"http://woodstock.temashdesign.com/watch/wp-content/uploads/sites/5/2016/03/woodstock-watch-logo-retina.png","id":"1863","height":"150","width":"494","thumbnail":"http://woodstock.temashdesign.com/watch/wp-content/uploads/sites/5/2016/03/woodstock-watch-logo-retina-150x150.png"},"tdl_site_logo_height":"75","td_logo_font":{"font-family":"Lato","font-options":"","google":"1","font-backup":"","font-weight":"700","font-style":"","subsets":"latin","text-transform":"","font-size":"40px","line-height":"40px","letter-spacing":"0px","color":"#333333"},"tdl_logo_description":"1","tdl_logo_tagline_font":{"font-family":"Lato","font-options":"","google":"1","font-backup":"","font-weight":"300","font-style":"","subsets":"latin","text-transform":"","font-size":"14px","line-height":"14px","letter-spacing":"0px","color":"#666666"},"td_logo_font_align":"center","tdl_favicon_image":{"url":"http://watch:8888/wp-content/themes/woodstock/favicon.png","id":"","height":"","width":"","thumbnail":""},"main_header_layout":"2","tdl_sticky_menu":"1","tdl_sticky_menu_hide":"1","tdl_sticky_menu_mobile":"1","tdl_header_search_bar":"1","tdl_header_ajax_search":"1","tdl_header_search_pt":"product","tdl_header_customer_bar":"1","tdl_header_customer_bar_title":"1-800-123-45-67","tdl_header_customer_bar_subtitle":"Customer Support","tdl_header_customer_bar_text":"[contact city=\"New York\" phone=\"1-800-333-42-63\" address=\"78 2nd House RD Montauk, NY, 11954\" email=\"ny@woodstock.com\"]\r\n[divider]\r\n[contact city=\"San Francisco\" phone=\"1-800-444-23-54\" address=\"1 Infinite Loop Cupertino, CA 95014\" email=\"sf@woodstock.com\"]\r\n[follow facebook=\"#\" twitter=\"#\" googleplus=\"#\" rss=\"#\"]","tdl_topbar_switch":"1","tdl_topbar_wishlist":"1","tdl_topbar_wpml":"1","tdl_topbar_social_icons":"1","tdl_uber_menu":"0","tdl_catalog_mode":"0","tdl_shop_breadcrumb":"1","tdl_sidebar_listing":"1","tdl_product_display_thumb":"standart","tdl_second_image_product_listing":"1","tdl_num_slider_images":"3","tdl_num_prevslider_images":"3","tdl_product_display_type":"grid","tdl_categories_per_column":"4","tdl_products_per_column":"4","tdl_product_count":"12,24,36","tdl_category_listing":"first_category","tdl_ratings_catalog_page":"1","tdl_product_description":"1","tdl_product_description_number":"20","tdl_sale_percentages":"1","tdl_salebadge_text":"Sale","tdl_newbadge":"0","tdl_newbadge_text":"New","tdl_newbadge_date":"5","tdl_out_of_stock_text":"Out of stock","tdl_product_sidebar":"3","tdl_product_gallery_zoom":"1","tdl_related_products_per_view":"4","tdl_sharing_options":"1","tdl_footer_layout":"4","tdl_footer_logos_off":"1","tdl_footer_logos":{"url":"http://woodstock.temashdesign.com/watch/wp-content/themes/woodstock/images/payment_cards.png","id":"","height":"","width":"","thumbnail":""},"tdl_footer_text":"&copy; 2016 - Woodstock Woocommerce Theme. Created by <a href='http://www.temashdesign.com'>TemashDesign</a>","tdl_main_color":"#cb9400","tdl_color_link":"#cb9400","tdl_header_ajax_loader":"spinner-bounce","tdl_page_loader_spinner":"4","tdl_page_loader_color":"#cb9400","tdl_page_loader_bg":"#1f1f1f","tdl_page_loader":"0","tdl_sticky_color_scheme":"sth-dark","tdl_sticky_background_color":"#1c1c1c","tdl_sticky_bgcolor_opacity":"100","tdl_stickydrop_color_scheme":"shd-dark","tdl_stickydrop_bgcolor":"#151515","tdl_stickydrop_bgcolor_opacity":"100","tdl_header_padding":"30","tdl_header_background_color":{"background-color":"#242424","background-repeat":"no-repeat","background-size":"cover","background-attachment":"","background-position":"center center","background-image":"http://woodstock.temashdesign.com/watch/wp-content/uploads/sites/5/2016/03/header-bg.jpg","media":{"id":"1865","height":"400","width":"1500","thumbnail":"http://woodstock.temashdesign.com/watch/wp-content/uploads/sites/5/2016/03/header-bg-150x150.jpg"}},"tdl_topbar_color_scheme":"td_dark","tdl_topbar_background_color":"#000000","tdl_topbar_background_opacity":"0","tdl_topbar_border":{"border-top":"1px","border-right":"1px","border-bottom":"1px","border-left":"1px","border-style":"solid","border-color":"#ffffff"},"tdl_topbar_border_opacity":"5","tdl_topbar_drop_color_scheme":"tbd_dark","tdl_topbar_dropdown_background_color":"#151515","tdl_topbar_dropdown_background_opacity":"100","tdl_header_search_icon":"e604","tdl_header_search_icon_color":"#999999","tdl_header_search_custom_icon":{"url":"","id":"","height":"","width":"","thumbnail":""},"tdl_header_searchbox_input_color":"#ffffff","tdl_header_searchbox_background_color":"#161616","tdl_header_searchbox_background_opacity":"0","tdl_header_searchbox_border_color":{"border-top":"1px","border-right":"1px","border-bottom":"1px","border-left":"1px","border-style":"solid","border-color":"#ffffff"},"tdl_header_searchbox_border_opacity":"10","tdl_header_searchboxdrop_color_scheme":"sd-dark","tdl_header_searchboxdrop_bgcolor_scheme":"#151515","tdl_header_searchboxdrop_bgcolor_opacity":"100","tdl_header_contactbox_icon":"e60b","tdl_header_customer_bar_icon_color":"#999999","tdl_header_customer_bar_icon":{"url":"","id":"","height":"","width":"","thumbnail":""},"tdl_header_customer_bar_color_scheme":"hc-dark","tdl_header_customer_bar_bgcolor":"#161616","tdl_header_customer_bar_bgcolor_opacity":"0","tdl_header_customer_bar_border_color":{"border-top":"1px","border-right":"1px","border-bottom":"1px","border-left":"1px","border-style":"solid","border-color":"#ffffff"},"tdl_header_customer_bar_border_opacity":"10","tdl_header_customerdrop_color_scheme":"csd-dark","tdl_header_customerdrop_bgcolor_scheme":"#161616","tdl_header_customerdrop_bgcolor_opacity":"100","tdl_header_shopcart_icon":"e603","tdl_header_shopcart_icon_color":"#cb9400","tdl_header_shopcart_custom_icon":{"url":"","id":"","height":"","width":"","thumbnail":""},"tdl_header_shopcart_color_scheme":"shc-dark","tdl_header_mobmenu_color_scheme":"mb-dark","tdl_header_mobmenu_bgcolor":"#000000","tdl_header_mobmenu_bgcolor_opacity":"0","tdl_header_mobmenu_border_color":{"border-top":"1px","border-right":"1px","border-bottom":"1px","border-left":"1px","border-style":"solid","border-color":"#ffffff"},"tdl_header_mobmenu_border_opacity":"10","tdl_mainnav_color_scheme":"mn-dark","tdl_mainnav_bgcolor":"#000000","tdl_mainnav_bgcolor_opacity":"0","tdl_mainnav_border":{"border-top":"1px","border-right":"1px","border-bottom":"1px","border-left":"1px","border-style":"solid","border-color":"#ffffff"},"tdl_mainnav_border_opacity":"5","tdl_mainnavdrop_color_scheme":"mnd-dark","tdl_mainnavdrop_bgcolor":"#161616","tdl_mainnavdrop_bgcolor_opacity":"100","tdl_title_bgcolor":"#1c1c1c","tdl_default_header_bg":{"url":"http://woodstock.temashdesign.com/watch/wp-content/uploads/sites/5/2016/03/mainshop-categories.jpg","id":"1866","height":"600","width":"1920","thumbnail":"http://woodstock.temashdesign.com/watch/wp-content/uploads/sites/5/2016/03/mainshop-categories-150x150.jpg"},"tdl_title_color_scheme":"mta-dark","tdl_title_align":"title-left","tdl_shop_header_parallax":"1","tdl_page_title_bgcolor":"#1c1c1c","tdl_page_default_header_bg":{"url":"","id":"","height":"","width":"","thumbnail":""},"tdl_page_title_color_scheme":"mta-dark","tdl_page_title_align":"title-left","tdl_page_header_parallax":"1","tdl_blog_title_bgcolor":"#1c1c1c","tdl_blog_default_header_bg":{"url":"","id":"","height":"","width":"","thumbnail":""},"tdl_blog_title_color_scheme":"mta-dark","tdl_blog_title_align":"title-left","tdl_blog_header_parallax":"1","tdl_content_bgcolor":"#242424","tdl_maincontent_color_scheme":"mc-dark","tdl_product_border":"no-border","tdl_product_border_color":"#3d3d3d","tdl_product_align":"palign-center","tdl_sidebarnav_color_scheme":"snd-dark","tdl_sidebarcart_color_scheme":"scd-dark","tdl_footer_bgcolor":"#1e1e1e","tdl_footer_color_scheme":"fc-dark","tdl_content_font":{"font-family":"Lato","font-options":"","google":"1","font-backup":"","font-weight":"300","font-style":"","subsets":"latin","font-size":"16px","line-height":"26px","letter-spacing":"0px"},"tdl_h1_heading_font":{"font-family":"Playfair Display","font-options":"","google":"1","font-backup":"","font-weight":"700","font-style":"","subsets":"latin","font-size":"36px","line-height":"50px","letter-spacing":"0px"},"tdl_h2_heading_font":{"font-family":"Playfair Display","font-options":"","google":"1","font-backup":"","font-weight":"700","font-style":"","subsets":"latin","font-size":"30px","line-height":"42px","letter-spacing":"0px"},"tdl_h3_heading_font":{"font-family":"Playfair Display","font-options":"","google":"1","font-backup":"","font-weight":"700","font-style":"","subsets":"latin","font-size":"24px","line-height":"34px","letter-spacing":"0px"},"tdl_h4_heading_font":{"font-family":"Playfair Display","font-options":"","google":"1","font-backup":"","font-weight":"400","font-style":"","subsets":"latin","font-size":"18px","line-height":"25px","letter-spacing":"0px"},"tdl_h5_heading_font":{"font-family":"Lato","font-options":"","google":"1","font-backup":"","font-weight":"700","font-style":"","subsets":"latin","font-size":"18px","line-height":"25px","letter-spacing":"0px"},"tdl_h6_heading_font":{"font-family":"Lato","font-options":"","google":"1","font-backup":"","font-weight":"300","font-style":"","subsets":"latin","font-size":"12px","line-height":"17px","letter-spacing":"3px"},"tdl_nav_font":{"font-family":"Playfair Display","font-options":"","google":"1","font-backup":"","font-weight":"700","font-style":"","subsets":"latin","text-transform":"capitalize","font-size":"18px","line-height":"24px","letter-spacing":"0px"},"tdl_subnav_font":{"font-family":"Lato","font-options":"","google":"1","font-weight":"300","font-style":"","subsets":"latin-ext","letter-spacing":"0px"},"twitter_link":"http://twitter.com/username","facebook_link":"https://www.facebook.com/username","googleplus_link":"","pinterest_link":"http://www.pinterest.com/","vimeo_link":"","youtube_link":"","flickr_link":"","skype_link":"","behance_link":"","dribble_link":"","tumblr_link":"","linkedin_link":"","github_link":"","vine_link":"","instagram_link":"","dropbox_link":"","rss_link":"","stumbleupon_link":"","paypal_link":"","foursquare_link":"","soundcloud_link":"","spotify_link":"","vk_link":"","android_link":"","apple_link":"","windows_link":"","tdl_blog_layout":"1","tdl_single_blog_layout":"1","tdl_blog_sharing_options":"1","tdl_custom_css":".mc-dark #products li.product-item figure.product-inner p.product-category-listing a,\r\n.site_header.with_featured_img .term-description p, .site_header.mta-dark .term-description p {\r\n    color:#cb9400 !important;\r\n}","tdl_custom_js_footer":"","opt-customizer-only":"2","redux-backup":1}

wp-content/plugins/jck-woo-quickview/inc/admin/settings.php
<?php
add_filter( 'wpsf_register_settings_jckqv', 'jckqv_settings' );

/**
 * Quickview Settings
 *
 * @param arr $wpsf_settings
 * @return arr
 */
function jckqv_settings( $wpsf_settings ) {

    $wpsf_settings = array(

        'tabs' => array(
            array(
                'id' => 'trigger',
                'title' => __('Quickview Trigger Settings', 'jckqv')
            ),
            array(
                'id' => 'popup',
                'title' => __('Popup Settings', 'jckqv')
            )
        ),

        'sections' => array(
            array(
                'tab_id' => 'trigger',
                'section_id' => 'general',
                'section_title' => 'General',
                'section_description' => '',
                'fields' => array(
                    array(
                        'id' => 'method',
                        'title' => __('Quickview Method', 'jckqv'),
                        'subtitle' => '',
                        'type' => 'select',
                        'default' => 'click',
                        'placeholder' => '',
                        'choices' => array(
                            'mouseover' => __('Hover Quickview Button', 'jckqv'),
                            'click' => __('Click Quickview Button', 'jckqv')
                        )
                    )
                )
            ),


            array(
                'tab_id' => 'trigger',
                'section_id' => 'position',
                'section_title' => 'Positioning',
                'section_description' => '',
                'fields' => array(
                    array(
                        'id' => 'autoinsert',
                        'title' => __('Automatically insert Button?', 'jckqv'),
                        'subtitle' => __('Would you like Quickview to attempt to automatically insert the Quickview button?<br><br> For alternative insertion options, please see <a href="http://www.jckemp.com/plugins/woocommerce-quickview/#manually_insert_the_quickview_button" target="_blank">the documentation</a>.', 'jckqv'),
                        'type' => 'checkbox',
                        'default' => 1,
                        'placeholder' => ''
                    ),
                    array(
                        'id' => 'position',
                        'title' => __('Button Position', 'jckqv'),
                        'subtitle' => __('If you chose to automatically insert the button, where should it be displayed?', 'jckqv'),
                        'type' => 'select',
                        'default' => 'beforetitle',
                        'placeholder' => '',
                        'choices' => array(
                            'beforeitem' => __('Before Item', 'jckqv'),
                            'beforetitle' => __('Before Title', 'jckqv'),
                            'aftertitle' => __('After Title', 'jckqv'),
                            'afteritem' => __('After Item', 'jckqv')
                        )
                    ),
                    array(
                        'id' => 'align',
                        'title' => __('Button Align', 'jckqv'),
                        'type' => 'select',
                        'default' => 'left',
                        'placeholder' => '',
                        'choices' => array(
                            'left' => __('Left', 'jckqv'),
                            'center' => __('Centre', 'jckqv'),
                            'right' => __('Right', 'jckqv'),
                            'none' => __('None', 'jckqv')
                        )
                    ),
                    array(
                        'id' => 'margins',
                        'title' => __('Margins', 'jckqv'),
                        'type' => 'multiinputs',
                        'subtitle' => __('Enter a pixel value (positive/negative) to offset the quickview button.', 'jckqv'),
                        'default' => array(
                            'Top' => 0,
                            'Right' => 0,
                            'Bottom' => 10,
                            'Left' => 0
                        ),
                        'placeholder' => '',
                    )
                )
            ),
            array(
                'tab_id' => 'trigger',
                'section_id' => 'styling',
                'section_title' => 'Styling',
                'section_description' => '',
                'fields' => array(
                    array(
                        'id' => 'autohide',
                        'title' => __('Auto-hide Button?', 'jckqv'),
                        'subtitle' => __('Should the quickview button only show when the product is hovered?', 'jckqv'),
                        'type' => 'checkbox',
                        'default' => 0,
                        'placeholder' => ''
                    ),
                    array(
                        'id' => 'hoverel',
                        'title' => __('Button Parent', 'jckqv'),
                        'type' => 'text',
                        'subtitle' => __('If the button is set to autohide, enter a parent class, id, or element that should display the quickview button when hovered.', 'jckqv'),
                        'default' => __('.product', 'jckqv'),
                        'placeholder' => '',
                    ),
                    array(
                        'id' => 'icon',
                        'title' => __('Icon', 'jckqv'),
                        'subtitle' => __('Choose the Icon to use on the Quickview button', 'jckqv'),
                        'type' => 'select',
                        'default' => 'eye',
                        'placeholder' => '',
                        'choices' => array(
                            'none' => __('No Icon', 'jckqv'),
                            'search' => __('Magnifier', 'jckqv'),
                            'eye' => __('Eye', 'jckqv'),
                            'plus' => __('Plus Symbol', 'jckqv')
                        )
                    ),
                    array(
                        'id' => 'text',
                        'title' => __('Quickview Text', 'jckqv'),
                        'type' => 'text',
                        'default' => __('Quickview', 'jckqv'),
                        'placeholder' => '',
                    ),
                    array(
                        'id' => 'btnstyle',
                        'title' => __('Button Style', 'jckqv'),
                        'type' => 'select',
                        'default' => 'flat',
                        'placeholder' => '',
                        'choices' => array(
                            'none' => __('None', 'jckqv'),
                            'border' => __('Border', 'jckqv'),
                            'flat' => __('Flat', 'jckqv')
                        )
                    ),
                    array(
                        'id' => 'padding',
                        'title' => __('Padding', 'jckqv'),
                        'type' => 'multiinputs',
                        'subtitle' => __('Enter a pixel value (positive/negative) to pad the quickview button.', 'jckqv'),
                        'default' => array(
                            'Top' => 8,
                            'Right' => 10,
                            'Bottom' => 8,
                            'Left' => 10
                        ),
                        'placeholder' => '',
                    ),
                    array(
                        'id' => 'btncolour',
                        'title' => __('Button Colour', 'jckqv'),
                        'subtitle' => __('If your button is "Flat" style, this is the background colour. If it\'s "Border" style, this is the border colour.', 'jckqv'),
                        'type' => 'color',
                        'default' => '#66cc99',
                        'placeholder' => '',
                    ),
                    array(
                        'id' => 'btnhovcolour',
                        'title' => __('Button Hover Colour', 'jckqv'),
                        'subtitle' => __('If your button is "Flat" style, this is the background colour on hover. If it\'s "Border" style, this is the border colour on hover.', 'jckqv'),
                        'type' => 'color',
                        'default' => '#47C285',
                        'placeholder' => '',
                    ),
                    array(
                        'id' => 'btntextcolour',
                        'title' => __('Button Text Colour', 'jckqv'),
                        'type' => 'color',
                        'default' => '#ffffff',
                        'placeholder' => '',
                    ),
                    array(
                        'id' => 'btntexthovcolour',
                        'title' => __('Button Text Hover Colour', 'jckqv'),
                        'type' => 'color',
                        'default' => '#ffffff',
                        'placeholder' => '',
                    ),
                    array(
                        'id' => 'borderradius',
                        'title' => __('Border Radius', 'jckqv'),
                        'type' => 'multiinputs',
                        'subtitle' => __('Enter a border radius in pixels.', 'jckqv'),
                        'default' => array(
                            'Top Left' => 4,
                            'Top Right' => 4,
                            'Btm Right' => 4,
                            'Btm Left' => 4
                        ),
                        'placeholder' => '',
                    ),
                )
            ),
            array(
                'tab_id' => 'popup',
                'section_id' => 'general',
                'section_title' => 'General',
                'section_description' => '',
                'fields' => array(
                    array(
                        'id' => 'gallery',
                        'title' => __('Enable Gallery?', 'jckqv'),
                        'subtitle' => __('Should the popup allow you to navigate between the other products on the page?', 'jckqv'),
                        'type' => 'checkbox',
                        'default' => 1,
                        'placeholder' => ''
                    ),
                    array(
                        'id' => 'overlaycolour',
                        'title' => __('Overlay Colour', 'jckqv'),
                        'type' => 'color',
                        'default' => '#000000',
                        'placeholder' => '',
                    ),
                    array(
                        'id' => 'overlayopacity',
                        'title' => __('Overlay Opacity', 'jckqv'),
                        'type' => 'text',
                        'subtitle' => __('Enter a value where 0 = transparent, and 1 = opaque.', 'jckqv'),
                        'default' => '0.8',
                        'placeholder' => '',
                    )
                )
            ),
            array(
                'tab_id' => 'popup',
                'section_id' => 'imagery',
                'section_title' => 'Imagery',
                'section_description' => '',
                'fields' => array(
                    array(
                        'id' => 'imgtransition',
                        'title' => __('Image Transition', 'jckqv'),
                        'type' => 'select',
                        'default' => 'horizontal',
                        'placeholder' => '',
                        'choices' => array(
                            'horizontal' => __('Horizontal Slide', 'jckqv'),
                            'fade' => __('Fade', 'jckqv')
                        )
                    ),
                    array(
                        'id' => 'transitionspeed',
                        'title' => __('Transition Speed', 'jckqv'),
                        'type' => 'text',
                        'subtitle' => __('The speed in milliseconds at which the image gallery transition occurs.', 'jckqv'),
                        'default' => '600',
                        'placeholder' => '',
                    ),
                    array(
                        'id' => 'autoplay',
                        'title' => __('Autoplay?', 'jckqv'),
                        'subtitle' => __('Automatically scroll through the product imagery?', 'jckqv'),
                        'type' => 'checkbox',
                        'default' => 0,
                        'placeholder' => ''
                    ),
                    array(
                        'id' => 'autoplayspeed',
                        'title' => __('Autoplay Speed', 'jckqv'),
                        'type' => 'text',
                        'subtitle' => __('The duration in milliseconds that each image is displayed.', 'jckqv'),
                        'default' => '3000',
                        'placeholder' => '',
                    ),
                    array(
                        'id' => 'infinite',
                        'title' => __('Infinite Scroll?', 'jckqv'),
                        'subtitle' => __('Infnitely scroll through the product imagery?', 'jckqv'),
                        'type' => 'checkbox',
                        'default' => 1,
                        'placeholder' => ''
                    ),
                    array(
                        'id' => 'navarr',
                        'title' => __('Show Navigation Arrows?', 'jckqv'),
                        'subtitle' => __('Show previous/next arrows on the image gallery?', 'jckqv'),
                        'type' => 'checkbox',
                        'default' => 1,
                        'placeholder' => ''
                    ),
                    array(
                        'id' => 'thumbnails',
                        'title' => __('Thumbnail type', 'jckqv'),
                        'type' => 'select',
                        'default' => 'thumbnails',
                        'placeholder' => '',
                        'choices' => array(
                            'thumbnails' => __('Sliding Thumbnails', 'jckqv'),
                            'bullets' => __('Bullets', 'jckqv'),
                            'none' => __('None', 'jckqv')
                        )
                    )
                )
            ),
            array(
                'tab_id' => 'popup',
                'section_id' => 'content',
                'section_title' => 'Content',
                'section_description' => '',
                'fields' => array(
                    array(
                        'id' => 'showtitle',
                        'title' => __('Show Title?', 'jckqv'),
                        'type' => 'checkbox',
                        'default' => 1,
                        'placeholder' => ''
                    ),
                    array(
                        'id' => 'showprice',
                        'title' => __('Show Price?', 'jckqv'),
                        'type' => 'checkbox',
                        'default' => 1,
                        'placeholder' => ''
                    ),
                    array(
                        'id' => 'showrating',
                        'title' => __('Show Rating?', 'jckqv'),
                        'type' => 'checkbox',
                        'default' => 1,
                        'placeholder' => ''
                    ),
                    array(
                        'id' => 'showbanner',
                        'title' => __('Show Banner?', 'jckqv'),
                        'subtitle' => __('E.g: "Sale!"', 'jckqv'),
                        'type' => 'checkbox',
                        'default' => 1,
                        'placeholder' => ''
                    ),
                    array(
                        'id' => 'showdesc',
                        'title' => __('Show Description?', 'jckqv'),
                        'type' => 'select',
                        'default' => 'full',
                        'placeholder' => '',
                        'choices' => array(
                            'no' => __('No', 'jckqv'),
                            'full' => __('Full', 'jckqv'),
                            'short' => __('Short', 'jckqv')
                        )
                    ),
                    array(
                        'id' => 'showatc',
                        'title' => __('Show Product Options / Add to Cart?', 'jckqv'),
                        'type' => 'checkbox',
                        'default' => 1,
                        'placeholder' => ''
                    ),
                    array(
                        'id' => 'ajaxcart',
                        'title' => __('Enable AJAX Add to Cart?', 'jckqv'),
                        'subtitle' => __('Add products to cart from the quickview without reloading the page.', 'jckqv'),
                        'type' => 'checkbox',
                        'default' => 0,
                        'placeholder' => ''
                    ),
                    array(
                        'id' => 'autohidepopup',
                        'title' => __('Hide Popup After Add to Cart?', 'jckqv'),
                        'subtitle' => __('Once a product has been added to the cart via ajax, hide the modal popup..', 'jckqv'),
                        'type' => 'checkbox',
                        'default' => 1,
                        'placeholder' => ''
                    ),
                    array(
                        'id' => 'showqty',
                        'title' => __('Show Qty Field?', 'jckqv'),
                        'type' => 'checkbox',
                        'default' => 1,
                        'placeholder' => ''
                    ),
                    array(
                        'id' => 'showmeta',
                        'title' => __('Show Product Meta?', 'jckqv'),
                        'subtitle' => __('E.g: Categories, Tags, etc.', 'jckqv'),
                        'type' => 'checkbox',
                        'default' => 1,
                        'placeholder' => ''
                    ),
                    array(
                        'id' => 'themebtn',
                        'title' => __('Try to use your Theme\'s Button Styling?', 'jckqv'),
                        'subtitle' => __('Check this if you\'d like to attempt to use the button styling from your theme. If this fails, use the styling tools below, or add your own CSS.', 'jckqv'),
                        'type' => 'checkbox',
                        'default' => 0,
                        'placeholder' => ''
                    ),
                    array(
                        'id' => 'btncolour',
                        'title' => __('Button Colour', 'jckqv'),
                        'type' => 'color',
                        'default' => '#66cc99',
                        'placeholder' => '',
                    ),
                    array(
                        'id' => 'btnhovcolour',
                        'title' => __('Button Hover Colour', 'jckqv'),
                        'type' => 'color',
                        'default' => '#47C285',
                        'placeholder' => '',
                    ),
                    array(
                        'id' => 'btntextcolour',
                        'title' => __('Button Text Colour', 'jckqv'),
                        'type' => 'color',
                        'default' => '#ffffff',
                        'placeholder' => '',
                    ),
                    array(
                        'id' => 'btntexthovcolour',
                        'title' => __('Button Text Hover Colour', 'jckqv'),
                        'type' => 'color',
                        'default' => '#ffffff',
                        'placeholder' => '',
                    ),
                )
            )
        )

    );

    return $wpsf_settings;

}